v3.10/getting-started/kubernetes/installation/calico.md

```diff
@@ -50,7 +50,7 @@ datastore type and number of nodes.
 1. Download the {{site.prodname}} networking manifest for the Kubernetes API datastore.
 
    ```bash
-   curl {{site.url}}/{{page.version}}/manifests/calico-typha.yaml -o calico.yaml
+   curl {{site.url}}/{{page.version}}/manifests/calico-typha.yaml -o calico-typha.yaml
```

Comment:
Instead of this change I think we should change the include statement below to be `yaml="calico"`. We also need a similar change to line 102 (in the etcd datastore section).

Reply:
hahaha，This is not the first time I've found such a simple mistake. It's very simple for experienced people, but it's not very friendly for beginners